app/src/main/res/values-en/strings.xml

```diff
@@ -0,0 +1,214 @@
+
+    <string name="diagnosis_locked_title">You have already reported the infection</string>
+    <string name="diagnosis_locked_body1">The information you provided will help others make appropriate decisions about their health.</string>
+    <string name="diagnosis_locked_body2">The app is now locked and will no longer send or collect encounter information. Thank you for using the app and helping fight COVID-19.</string>
+    <string name="diagnosis_locked_image_description">Locked</string>
+
+    <string name="diagnosis_instructions_1">The unlock code cannot be used to identify you.</string>
+    <string name="diagnosis_instructions_2">Corona Alert notifies users who have been in your vicinity for some duration of infection risk.</string>
+    <string name="diagnosis_instructions_3">This information encourages recipients to avoid contact, monitor their symptoms and take a test if necessary, helping avoid further spread of infection.</string>
+
+    <string name="code_entry_title">Enter unlock code</string>
+    <string name="code_entry_invalid">The unlock code is invalid. Please check that you entered the code correctly.</string>
+    <string name="code_entry_submit_failed">Your infection could not be reported because the server could not be reached. Please try again in a moment.</string>
+    <string name="code_entry_en_disabled">Exposure notifications are off. Return to the home view where you can turn them on.</string>
+    <string name="code_entry_info">The unlock code cannot be used to identify you.</string>
+
+    <string name="diagnosis_complete_title">Thank you for your message</string>
+    <string name="diagnosis_complete_body">The information you provide will help control the coronavirus epidemic. The app is now locked and no longer collects information about encounters. Get well soon!</string>
+    <string name="diagnosis_complete_continue">Continue</string>
+
+    <string name="home_title">Home</string>
+    <string name="home_exposure_label">Potential exposure</string>
+    <string name="home_no_exposure_label">No exposures observed</string>
+    <string name="home_exposure_sub_label">You may have been exposed to the virus</string>
+    <string name="home_no_exposure_sub_label">You have not been near a person who reported an infection</string>
+    <string name="home_exposure_instructions">See instructions</string>
+
+    <string name="home_symptom_label">Do you suspect an infection?</string>
+    <string name="home_prevention_label">This is how you protect yourself</string>
+    <string name="home_prevention_url">https://thl.fi/en/web/infectious-diseases-and-vaccinations/what-s-new/coronavirus-covid-19-latest-updates/transmission-and-protection-coronavirus</string>
+    <string name="home_statistics_label">Coronavirus situation in Finland</string>
+    <string name="home_statistics_url">https://thl.fi/en/web/infectious-diseases-and-vaccinations/what-s-new/coronavirus-covid-19-latest-updates</string>
```

Comment:
This is the EN version of FI https://thl.fi/fi/web/infektiotaudit-ja-rokotukset/ajankohtaista/ajankohtaista-koronaviruksesta-covid-19/tilannekatsaus-koronaviruksesta

```suggestion
    <string name="home_statistics_url">https://thl.fi/en/web/infectious-diseases-and-vaccinations/what-s-new/coronavirus-covid-19-latest-updates/situation-update-on-coronavirus</string>
```